src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -251,6 +251,7 @@ private static void removeTmp(Path tmp) throws IOException {
             @Override
             public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
                 throws IOException {
+              file.toFile().setWritable(true);
```

Comment:
This fixes `java.nio.file.AccessDeniedException: bazel-out\x64_windows-fastbuild\...` on Windows

Reply:
wrapped with `isWindows`